doc/release-notes.md

```diff
@@ -4,6 +4,45 @@ release-notes at release time)
 Notable changes
 ===============
 
+Network Upgrade 4: Canopy
+--------------------------
+
+The code preparations for the Canopy network upgrade are finished and included in this release. The following ZIPs are being deployed:
+
+- [ZIP 207: Funding Streams](https://zips.z.cash/zip-0207)
+- [ZIP 211: Disabling Addition of New Value to the Sprout Value Pool](https://zips.z.cash/zip-0211)
+- [ZIP 212: Allow Recipient to Derive Sapling Ephemeral Secret from Note Plaintext](https://zips.z.cash/zip-0212)
+- [ZIP 214: Consensus rules for a Zcash Development Fund](https://zips.z.cash/zip-0214)
+- [ZIP 215: Explicitly Defining and Modifying Ed25519 Validation Rules](https://zips.z.cash/zip-0215)
+
+Canopy will activate on testnet at height 950000, and can also be activated at a specific height in regtest mode by setting the config option `-nuparams=0xe9ff75a6:HEIGHT`.
```

Comment:
We're already past that height on Testnet.  The current Testnet height at 2020-07-13 10:00 UTC is 1000068. According to [the schedule](https://time.graphics/line/359234), NU4 should activate on Testnet at the end of July / start of August. August 1st at 20:20 UTC would be around block 1022500.

Reply:
Oh, August 1st is a Saturday. That's probably not a good idea.